### PR TITLE
UK TL: 95% translated

### PR DESCRIPTION
**Pull request by tsunamistate**
I wanted to make a PR when I got to 100%, but seeing that 1.4.0 is releasing soon, decided to just **DO IT** now

@SourceCodeDeleted please review this when you have the time

### CODE REVIEW

**Comment by AJenbo** [comment]
Good choice, :)

File: Translations/uk.po [hunk 1]
```diff
@@ -10533,6 +10816,7 @@ msgid ""
 "Look.  I'm a cow.  And you, you're monster bait. Get some experience under "
 "your belt!  We'll talk..."
 msgstr ""
+"Подивись.  Я корова. А ти, ти приманка. Наберись досвіду!  Тоді і поговоримо…"
```
Comment (AJenbo) [inline Translations/uk.po:10819]:
```suggestion
"Подивись. Я корова. А ти, ти приманка. Наберись досвіду!  Тоді і поговоримо…"
```

**Comment by AJenbo** [comment]
@SourceCodeDeleted says he won't be able to review this before tomorrow. If no one else does so before then I probably going to merge this as the release is likely to go out before tomorrow.

**Comment by SourceCodeDeleted** [comment]
Ok this looks good . Nothing needed to change that I can see.

**Pull request merged**